### PR TITLE
Make toolbar padding 16px

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
-  This changes the content toolbar padding from 24px to 16px (light/right).

There's a pending question that I have regarding the menu icon & title spacing in the angular-design-patterns PR.   This question applies here too.

### CODE REVIEW

**Review by joebochill — COMMENTED** [review]
Annoying, I know, but would prefer if we did this in JSS with each example so that the patterns stay isolated (copy-pastable) without shared resources/styles.

**Comment by emclaug2** [comment]
> Annoying, I know, but would prefer if we did this in JSS with each example so that the patterns stay isolated (copy-pastable) without shared resources/styles.

Geez this comment also applies to the angular PR equivalent.    I'll adjust the styles in the other PR as well.  No shared/common classes/styles.

**Review by joebochill — APPROVED** [review]
Collapsible Appbar (large version, mobile viewport) menu icon is off. Otherwise 🔥

**Comment by emclaug2** [comment]
> Collapsible Appbar (large version, mobile viewport) menu icon is off. Otherwise 🔥

I'm not sure where it is intended to go; the padding for the toolbar is correct at 16px.

**Comment by huayunh** [comment]
https://material.io/components/app-bars-top#specs @emclaug2 

see " Extended top app bar"

**Comment by jeffvg** [comment]
does anyone see this console warning in this branch? it might be from dev but not sure at the moment.

Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    in LoadingStates (created by Context.Consumer)

**Comment by joebochill** [comment]
image

Toolbar has padding: 24
Icon Button has padding: 12, margin -12

Total distance from left edge to icon is: 24+12-12 = 24 (should be 16)

**Review by jeffvg — APPROVED** [review]

**Review by joebochill — CHANGES_REQUESTED** [review]
See comment above about the collapsible appbar spacing

File: src/pages/app-bar/collapsible-app-bar/index.tsx [hunk 1]
```diff
@@ -11,6 +11,12 @@ import { InfoListItem } from '@pxblue/react-components';
 const MAX_APP_BAR_HEIGHT = 128; // Specified by Material Design
 
 const useStyles = makeStyles((theme: Theme) => ({
+    appbarRoot: {
+        padding: 0,
```
Comment (joebochill) [inline src/pages/app-bar/collapsible-app-bar/index.tsx:15]: Is this actually doing anything?

File: src/pages/app-bar/collapsible-app-bar/index.tsx [hunk 2]
```diff
@@ -125,7 +132,7 @@ export const CollapsibleAppBar = (): JSX.Element => {
                 </Toolbar>
             </AppBar>
             <div className={clsx(classes.banner, classes.alignTopContent)}>
-                <Toolbar className={classes.bannerMain} data-cy={'banner'}>
+                <Toolbar className={classes.bannerMain} data-cy={'banner'} style={{ paddingLeft: 16 }}>
```
Comment (joebochill) [inline src/pages/app-bar/collapsible-app-bar/index.tsx:135]: classes={{ gutters: classes.toolbarGutters }}

File: src/pages/app-bar/collapsible-app-bar/index.tsx [hunk 3]
```diff
@@ -134,7 +141,7 @@ export const CollapsibleAppBar = (): JSX.Element => {
                                 onClick={(): void => {
                                     dispatch({ type: TOGGLE_DRAWER, payload: true });
                                 }}
-                                edge={'start'}
+                                style={{ marginRight: 20, marginLeft: -12 }}
```
Comment (joebochill) [inline src/pages/app-bar/collapsible-app-bar/index.tsx:144]:
```suggestion
                                edge={'start'}
                                style={{ marginRight: 20 }}
```

**Review by joebochill — APPROVED** [review]

File: src/pages/app-bar/collapsible-app-bar/index.tsx (hunk 2 above)
Comment (emclaug2) [reply src/pages/app-bar/collapsible-app-bar/index.tsx]: Good catch, this is updated now

File: src/pages/app-bar/collapsible-app-bar/index.tsx (hunk 1 above)
Comment (emclaug2) [reply src/pages/app-bar/collapsible-app-bar/index.tsx:15]: Yeah this is for the normal toolbar, not the collapsible one.

**Pull request merged**